### ISSUE TITLE
"retryTapIfNoChange" enhancement？

### ISSUE THREAD

**Issue by Ashen-You** [mobile-dev-inc/Maestro#1465]
The Android APP is in Flutter debug mode, which frequently triggers double taps. Can you provide a global configuration, or provide a custom command way for us to write the flow command we want easier than runFlow

**Comment by bartekpacia** [comment]
Hi @Ashen-You, what do you mean by this:

> The Android APP is in Flutter debug mode, which frequently triggers double taps.

Could you show an example of this behavior (or better: provide a reproducible mini-project demonstrating it)?

**Comment by Ashen-You** [comment]
> I guess that's because the app runs so slow in debug mode, right?
> 
> Linking for reference: [[Android] Accidental double tap](https://maestro.mobile.dev/troubleshooting/known-issues#android-accidental-double-tap)

Yes, this caused me to add 'retryTapIfNoChange' to almost all of tapOns. It would be nice to have global control
